solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1542 @@
+    );
+
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovingFundsTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movingFundsTimeoutNotifierRewardMultiplier
+    );
+
+    event MovedFundsSweepTxMaxTotalFeeUpdateStarted(
+        uint64 newMovedFundsSweepTxMaxTotalFee,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTxMaxTotalFeeUpdated(
+        uint64 movedFundsSweepTxMaxTotalFee
+    );
+
+    event MovedFundsSweepTimeoutUpdateStarted(
+        uint32 newMovedFundsSweepTimeout,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutUpdated(uint32 movedFundsSweepTimeout);
+
+    event MovedFundsSweepTimeoutSlashingAmountUpdateStarted(
+        uint96 newMovedFundsSweepTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutSlashingAmountUpdated(
+        uint96 movedFundsSweepTimeoutSlashingAmount
+    );
+
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovedFundsSweepTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movedFundsSweepTimeoutNotifierRewardMultiplier
+    );
+
+    event WalletCreationPeriodUpdateStarted(
+        uint32 newWalletCreationPeriod,
+        uint256 timestamp
+    );
+    event WalletCreationPeriodUpdated(uint32 walletCreationPeriod);
+
+    event WalletCreationMinBtcBalanceUpdateStarted(
+        uint64 newWalletCreationMinBtcBalance,
+        uint256 timestamp
+    );
+    event WalletCreationMinBtcBalanceUpdated(
+        uint64 walletCreationMinBtcBalance
+    );
+
+    event WalletCreationMaxBtcBalanceUpdateStarted(
+        uint64 newWalletCreationMaxBtcBalance,
+        uint256 timestamp
+    );
+    event WalletCreationMaxBtcBalanceUpdated(
+        uint64 walletCreationMaxBtcBalance
+    );
+
+    event WalletClosureMinBtcBalanceUpdateStarted(
+        uint64 newWalletClosureMinBtcBalance,
+        uint256 timestamp
+    );
+    event WalletClosureMinBtcBalanceUpdated(uint64 walletClosureMinBtcBalance);
+
+    event WalletMaxAgeUpdateStarted(uint32 newWalletMaxAge, uint256 timestamp);
+    event WalletMaxAgeUpdated(uint32 walletMaxAge);
+
+    event WalletMaxBtcTransferUpdateStarted(
+        uint64 newWalletMaxBtcTransfer,
+        uint256 timestamp
+    );
+    event WalletMaxBtcTransferUpdated(uint64 walletMaxBtcTransfer);
+
+    event WalletClosingPeriodUpdateStarted(
+        uint32 newWalletClosingPeriod,
+        uint256 timestamp
+    );
+    event WalletClosingPeriodUpdated(uint32 walletClosingPeriod);
+
+    event FraudChallengeDepositAmountUpdateStarted(
+        uint256 newFraudChallengeDepositAmount,
+        uint256 timestamp
+    );
+    event FraudChallengeDepositAmountUpdated(
+        uint256 fraudChallengeDepositAmount
+    );
```

Comment:
The type of `fraudChallengeDepositAmount` is `uint96`:
```
uint96 fraudChallengeDepositAmount;
```

Let's change it in every occurrence (events, functions).